src/ts/index.ts

```diff
@@ -393,3 +393,21 @@ export function signOrder(
       );
   }
 }
+
+/**
+ * Compute the unique identifier describing a user order in the settlement
+ * contract.
+ *
```

Comment:
I like this formatting a lot! Maybe we can update other functions to also add an empty ` *` line between descript and `@param`-eters whenever they get touched.